apps.yaml

```
@@ -12,11 +12,9 @@ aiidalab-optimade:
   - utilities
   logo: https://raw.githubusercontent.com/CasperWA/voila-optimade-client/232f11d39076b328e07c8bb6928c3a6847523f6c/img/optimade-no-text-transparent-bg.png
   metadata:
-    # TODO: Switch URL to "develop" branch after testing.
-    $ref: https://raw.githubusercontent.com/CasperWA/voila-optimade-client/integrate-aiidalab-app/.aiidalab/metadata.yaml
+    $ref: https://raw.githubusercontent.com/CasperWA/voila-optimade-client/develop/.aiidalab/metadata.yaml
   releases:
-    # TODO: Update branch after testing.
-  - "git+https://github.com/CasperWA/voila-optimade-client.git@integrate-aiidalab-app:2021.8.4rc0+aiidalab^.."
+  - "git+https://github.com/CasperWA/voila-optimade-client.git@2021.8.6^.."
```

I didn't notice this before, but there is a slight error here (the `:` indicates that this is a release line, not a single release):
```suggestion
  - "git+https://github.com/CasperWA/voila-optimade-client.git@:2021.8.6^.."
```
The value before the `:` indicates the branch, which when left empty is implicitly the default branch. While not strictly  necessary, I'd actually recommend to be explicit about the branch here:
```suggestion
  - "git+https://github.com/CasperWA/voila-optimade-client.git@develop:2021.8.6^.."
```
I will leave it up to you which one you prefer.